src/control/server/storage/scm/ipmctl.go

```diff
@@ -201,27 +201,24 @@ func (cr *cmdRunner) handleFreeCapacity(sockSelector int, nrNsPerSock uint, regi
 		return nil, nil, errors.Wrap(err, "createNamespaces")
 	}
 
-	numaSelector := sockAny
 	switch len(numaIDs) {
 	case 0:
-		return nil, nil, errors.New("no numa nodes were processed")
+		return nil, nil, errors.New("no namespaces created on regions with free capacity")
 	case 1:
```

Comment:
Nit: not sure it still needed.
If yes, the switch could probably be replace by a simple `if len(numaIDs) == 0`